src/reference/workers/taskcluster-worker/engine-plugin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Engine-Plugin interaction
+order: 40
+sequence_diagrams: true
+---
+
+Putting this all together, we get the following interactions.
```

Comment:
With the previous page being split up into separate pages, I fear we lose the intentional sequencing we had before. For example, this page starts with "Putting this all together" - which does not make sense if you didn't read the other pages first.

Is the original sequencing preserved? I kind of liked it being on a single page, because it incrementally introduced concepts, so the individual sections were not references that should be read out of context of the others, rather the entire content was meant to be a single self-contained chapter (much like you wouldn't split up a single blog post into separate reference pages to be read independently from one another).

Can we revert this back to a single page?

Thanks!